src/interface/interface.ts

```diff
@@ -21,6 +21,9 @@ export interface NodeStatistics {
     setup_timing_stats?: TimingStatistics;
     proof_timing_stats?: TimingStatistics;
   };
+  latest_version_info?: {
```

Comment:
This is for recording prover node's version?
If yes, we should remove the "latest"? just version_info? Any reason we add "latest"?